src/main/java/az/ingress/ms_order/queue/QueueSender.java

```diff
@@ -0,0 +1,19 @@
+package az.ingress.ms_order.queue;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.amqp.core.AmqpTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class QueueSender {
```

Comment:
burda abstraction/concrete istifade etsen yuxaridaki kimi QueueName-i servis-in bilmeyine ehtiyac qalmayacaq, ex:

```
QueueProducer
NotificationQueueProducer
```